project/Dependencies.scala

```diff
@@ -41,7 +41,7 @@ object Dependencies {
   object Compile {
     // Compile
 
-    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.17.7" exclude("org.slf4j", "slf4j-api") // ApacheV2
+    val camelCore     = "org.apache.camel"            % "camel-core"                   % "2.19.2" exclude("org.slf4j", "slf4j-api") // ApacheV2
```

Comment:
This looks too aggressive for a change, have you tried things with this bump? We only recently jumped to 17 because of a security think I believe?

Reply:
changing back this one and force pushing.